var/spack/repos/builtin/packages/libxcrypt/package.py

```diff
@@ -22,6 +22,8 @@ def url_for_version(self, version):
 
     license("LGPL-2.1-or-later")
 
+    version("4.4.38", sha256="80304b9c306ea799327f01d9a7549bdb28317789182631f1b54f4511b4206dd6")
```

Comment:
Suggestion to add comment on why 4.4.37 is skipped.
```suggestion
    version("4.4.38", sha256="80304b9c306ea799327f01d9a7549bdb28317789182631f1b54f4511b4206dd6")
    # 4.4.37 requires pkg-config and is not included here
```